contribution/minikube/start_minikube.sh

```diff
@@ -3,9 +3,11 @@
 # Copyright 2021 Authors of KubeArmor
 
 # Refer to https://minikube.sigs.k8s.io/docs/tutorials/ebpf_tools_in_minikube
+# download minikube iso
+wget -O minikube.iso https://github.com/kubearmor/kastore/blob/main/iso/minikube/minikube.iso?raw=true
```

Comment:
is this a custom image? is this based on kernel 4.19.94?

Reply:
Yes, its a custom image based on 4.19.94. We have a new repo named `kastore`